katsdpimageutils/primarybeamcorrection/primary_beam_correction.py

```diff
@@ -90,25 +97,27 @@ def central_freq(path):
 
 
 def cosine_power_pattern(separation_rad, c_freq):
-    """Compute Power patterns for a given frequency based on the
-    Cosine-squared power approximation from Mauch et al. (2020).
-    
+    """Compute Power patterns for a given frequency.
+
+    This uses the Cosine-squared power approximation from
+    Mauch et al. (2020).
+
     Parameters
     ----------
-    separation_rad: numpy array
+    separation_rad : numpy array
         Radial separation array
-    c_freq: numpy array
+    c_freq : numpy array
         An array of central frequencies for each frequency plane
     """
     rho = separation_rad
-    # Degrees to radians conversion
+    # convert degrees to radians
```

Comment:
You don't really need this comment any more, since the function name (deg2rad) already tells one that (whereas in the conversion from GHz to Hz below, it's not immediately obvious what the units are from the code.

Reply:
Noted and fixed.